First commit

brellin opened the pull request:
Chose a template, replaced header and title with my name, replaced main bg, and worked on intro and work ids.

brellin committed:
Chose a template, replaced header and title with my name, replaced main bg, and worked on intro and work ids

jthomps838 commented:
Great job! Are there any new CSS properties that you came across while you viewed the source code?

brellin commented:
There's so much that I saw on this thing that I have no idea how it works, but there is one thing that I implemented that wasn't part of it that I think really makes it pop: background-attachment: fixed
